xmtp_mls/src/groups/group_permissions.rs

```diff
@@ -294,29 +552,78 @@ impl PolicySet {
         })
     }
 
+    // In case group creator is on future version of libxmtp, we can validate
```

Comment:
This function enables users to validate metadata updates for metadata fields that might not exist in their version of libxmtp